docs/DeveloperGuide.md

```diff
@@ -541,13 +611,19 @@ Use case ends.
 
 Use case ends.
 
+* 3a. User provides incorrectly formatted information.
+* 3b. Cobb shows an error message.
```

Comment:
Correct me if i'm wrong, but shouldn't the extension number be on the step that was last executed, instead of being on the proceeding step? In this case it should be 2a and 2b (since step 3 will not be executed due to the error), while the previous lines should be 1a and 1b.